File: test-schemas-domain-classes/src/main/scala/testdomains/gratefuldead/EdgeTypes.scala
package testdomains.gratefuldead.edges

object Followedby {
  val Label                        = "followedBy"
  val propertyName: Option[String] = Some("weight")
}

class Followedby(src_4762: flatgraph.GNode, dst_4762: flatgraph.GNode, subSeq_4862: Int, property_4862: Any)
    extends flatgraph.Edge(src_4762, dst_4762, 0.toShort, subSeq_4862, property_4862) {
  override def propertyName: Option[String] = Followedby.propertyName
}

object Sungby {
  val Label = "sungBy"

}

class Sungby(src_4762: flatgraph.GNode, dst_4762: flatgraph.GNode, subSeq_4862: Int, property_4862: Any)
    extends flatgraph.Edge(src_4762, dst_4762, 1.toShort, subSeq_4862, property_4862) {}

object Writtenby {
  val Label = "writtenBy"

}

class Writtenby(src_4762: flatgraph.GNode, dst_4762: flatgraph.GNode, subSeq_4862: Int, property_4862: Any)
    extends flatgraph.Edge(src_4762, dst_4762, 2.toShort, subSeq_4862, property_4862) {}

File: test-schemas-domain-classes/src/main/scala/testdomains/gratefuldead/EdgeTypes.java
package testdomains.gratefuldead;

import java.util.HashSet;
import java.util.Set;

public class EdgeTypes {


public static final String FOLLOWEDBY = "followedBy";


public static final String SUNGBY = "sungBy";


public static final String WRITTENBY = "writtenBy";

public static Set<String> ALL = new HashSet<String>() {{
add(FOLLOWEDBY);
add(SUNGBY);
add(WRITTENBY);
}};

}